<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Breather Settings</string>
  </property>
  <property name="windowOpacity">
   <double>0.750000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>621</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="InhaleShape">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Shape</string>
    </property>
    <widget class="QRadioButton" name="InhEllipse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ellipse</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhRectangle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhRoundedRect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rounded Rect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="InhalePosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Position</string>
    </property>
    <widget class="QRadioButton" name="InhtopLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Top Left</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhtopCentre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Top Centre</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhtopRight">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>30</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Top Right</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="Inhcentre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Centre</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhcentreRight">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Right</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhcentreLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Left</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhbottomRight">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment=" ">Bottom Right</string>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhbottomLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Bottom Left</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhbottomCentre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Bottom Centre</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="InhaleChangable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Which direction should the shape increase"/>
    </property>
    <property name="title">
     <string>Direction</string>
    </property>
    <widget class="QRadioButton" name="InhOnlyHorizontal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Only Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhOnlyVertical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Only Vert</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="InhBothChangable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Both changable</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="HoldShape">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Shape</string>
    </property>
    <widget class="QRadioButton" name="HoldEllipse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ellipse</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldRectangle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldRoundedRect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rounded Rect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="HoldPosition">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>135</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Position</string>
    </property>
    <widget class="QRadioButton" name="HoldtopLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Top Left</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldtopCentre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Top Centre</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldtopRight">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>30</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Top Right</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="Holdcentre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Centre</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldcentreRight">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Right</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldcentreLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Left</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldbottomRight">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Bottom Right</string>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldbottomLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Bottom Left</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldbottomCentre">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Bottom Centre</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="HoldChangable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Which direction should the shape increase"/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="title">
     <string>Direction</string>
    </property>
    <widget class="QRadioButton" name="HoldOnlyHorizontal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Only Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldOnlyVertical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Only Vert</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="HoldBothChangable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Both changable</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonReset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>423</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Default</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDiscard">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>423</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Discard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>423</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Inhale">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>134</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inhale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Exhale">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>164</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Exhale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HoldIn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>134</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HoldOut">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>164</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold Out</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="timeInhale">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Inhale time (seconds)</string>
    </property>
    <property name="maximum">
     <double>65.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="timeExhale">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Exhale time (seconds)</string>
    </property>
    <property name="maximum">
     <double>65.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="timeHoldOut">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Hold Out Time (seconds)</string>
    </property>
    <property name="maximum">
     <double>65.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="timeHoldIn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Hold In time (seconds)</string>
    </property>
    <property name="maximum">
     <double>65.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="ChangeColour">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Colour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorSetHoldOut">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorSetExhale">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exhale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorSetHoldIn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorSetInhale">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Inhale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Show Help</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChangeTime">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HoldHz">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>395</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PosInExVert">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>365</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PosInExHz">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>395</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PosHoldVert">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>365</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeInhVertical">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>361</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Vertical Shape (0-1000)</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeInhHorizontal">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>390</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Horizontal Shape (0-1000)</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeHoldInHorizontal">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>390</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Horizontal Shape (0-1000)</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeHoldInVertical">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>361</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Vertical Shape (0-1000)</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_SetTransparancyShape">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape Transparancy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SetTransparancyWindow">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Transparancy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="transparancyWindow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>246</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>set Window Transparancy %</string>
    </property>
    <property name="maximum">
     <number>80</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="transparancyShape">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>219</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>set Shape Transparancy %</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInhExh">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>22</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inhale / Exhale</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHold">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>22</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hold Mode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
